.github/workflows/fix.yml

```
@@ -1,38 +1,38 @@
+#             🎨 yarn run prettier
 
-            [dependabot skip]
-          skip-empty: true
+#             [dependabot skip]
+#           skip-empty: true
 
-      - if: ${{ steps.actions--create-commit.outputs.commit }}
-        run: git push
\ No newline at end of file
+#       - if: ${{ steps.actions--create-commit.outputs.commit }}
+#         run: git push
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)